app/models/offer.rb

```diff
@@ -1,8 +1,20 @@
 class Offer < ApplicationRecord
   belongs_to :user
+
+  include PgSearch::Model
+  pg_search_scope :global_search,
+    against: [ :title, :description ],
```

Comment:
We should category as well for the search.

Reply:
done